stable/admin/templates/_helpers.tpl

```diff
@@ -56,6 +56,53 @@ Also, we can't use a single if because lazy evaluation is not an option
 {{- end -}}
 {{- end -}}
 
+{{/*
+Return the proper insights image name
+*/}}
+{{- define "insights.image" -}}
+{{- $registryName := $.Values.insights.image.registry -}}
+{{- $repositoryName := $.Values.insights.image.repository -}}
+{{- $tag := $.Values.insights.image.tag | toString -}}
+{{/*
+Helm 2.11 supports the assignment of a value to a variable defined in a different scope,
+but Helm 2.9 and 2.10 doesn't support it, so we need to implement this if-else logic.
+Also, we can't use a single if because lazy evaluation is not an option
+*/}}
+{{- if .Values.global }}
+    {{- if .Values.global.imageRegistry }}
+        {{- printf "%s/%s:%s" .Values.global.imageRegistry $repositoryName $tag -}}
+    {{- else -}}
+        {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+    {{- end -}}
+{{- else -}}
+    {{- printf "%s/%s:%s" $registryName $repositoryName $tag -}}
+{{- end -}}
+{{- end -}}
+
+{{/*
+Return the proper insights image name
```

Comment:
The description is the same as for `insights.image` helper. The logic seems to be almost the same.
Maybe you can consider parametrizing it or invoking it with a different scope?
Same applies to `stable/database/templates/_helpers.tpl`.